.env.example

```diff
@@ -26,3 +26,7 @@ export REACT_APP_URL='app.papercups.io'
 
 # General
 export BACKEND_URL='app.papercups.io'
+
+# Customer.io
+export CUSTOMER_IO_SITE_ID='0000000000000000000'
+export CUSTOMER_IO_API_KEY='xxxxxxxxxxxxx'
```

Comment:
nit - let's add a newline to the end of the file to make that red thing go away :P